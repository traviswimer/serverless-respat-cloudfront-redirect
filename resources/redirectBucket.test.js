const redirectBucket = require('./redirectBucket');

describe('redirectBucket()', () => {
	test('generates bucket', () => {
		expect(redirectBucket({
			prefix: "ThePrefix",
			pattern_name: "ThePatternName",
			redirect_from: "TheRedirectFrom",
			redirect_to: "TheRedirectTo"
		})).toMatchSnapshot();
	});
});
