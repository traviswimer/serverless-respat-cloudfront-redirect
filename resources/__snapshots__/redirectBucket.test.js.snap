// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redirectBucket() generates bucket 1`] = `
Object {
  "Properties": Object {
    "BucketName": "TheRedirectFrom",
    "WebsiteConfiguration": Object {
      "RedirectAllRequestsTo": Object {
        "HostName": "TheRedirectTo",
        "Protocol": "https",
      },
    },
  },
  "Type": "AWS::S3::Bucket",
}
`;
